### ISSUE TITLE
Another bug..

### ISSUE THREAD

**Issue by 7wgs0gr04v** [M2Team/NSudo#23]
I found big bug in NSudo. It doesn't show full files.

To reproduce:
![Bug](https://user-images.githubusercontent.com/10778276/54531291-3595d280-4985-11e9-837a-eba6f32b3c68.png)

```
Put nsudo.exe in c:\windows\system32\oobe

[HKEY_LOCAL_MACHINE\SYSTEM\Setup]
"OsLoaderPath"="\"
"RestartSetup"=dword:00000000
"SetupType"=dword:00000002
"SystemPartition"="\Device\HarddiskVolume1"
"SystemSetupInProgress"=dword:00000001
"SetupPhase"=dword:00000004
"CmdLine"="c:\\windows\\system32\\cmd.exe"
"OOBEInProgress"=dword:00000000
"WorkingDirectory"="c:\windows\panther"
```

**Comment by MouriNaruto** [comment]
Got it and I will debug it.

**Comment by optomux** [comment]
It seems fixed in latest 7.0? Looks like it.

**Comment by MouriNaruto** [comment]
Yes, I will fix it in 7.0 if I found it.

**Comment by MouriNaruto** [comment]
The issue is opened too long. I don't know the issue whether I solved or not. So I closed this issue. Please fire a new issue if you still met the issue.
